typings/docker-toolbelt.d.ts

```diff
@@ -0,0 +1,27 @@
+declare module 'docker-toolbelt' {
```

Comment:
Can you put these types directly in docker-toolbelt please since we control that repo and it would be beneficial for everyone that way

Reply:
It's probably still an improvement even then

Reply:
I just checked and the types are complete, not sure why I didn't add it, but I'll do that now.